server/container/guice/guice-common/src/main/java/org/apache/james/JamesServerMain.java

```diff
@@ -19,7 +19,13 @@
 
 package org.apache.james;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public interface JamesServerMain {
+
+    Logger LOGGER = LoggerFactory.getLogger(JamesServerMain.class);
```

Comment:
what about defining a org.apache.james.CONFIGURATION logger to avoid spreading logs about configuration loading in different loggers?

Reply:
I think it makes sense. 
I made the change and added some developer docs around logging. 
It's meant as a starting point.

```
es_1  | 
james_1  | 19:37:08.840 [INFO ] o.a.j.CONFIGURATION - Loading configuration Basic{configurationPath=ConfigurationPath{value=file://conf/}, directories=JamesServerResourceLoader{rootDirectory=/opt/apache-james/james-server, varDirectory=/opt/apache-james/james-server/var/, confDirectory=/opt/apache-james/james-server/conf/}}
james_1  | WARNING: An illegal reflective access operation has occurred
james_1  | WARNING: Illegal reflective access by com.google.inject.internal.cglib.core.$ReflectUtils$1 (file:/opt/apache-james/james-server/lib/guice-4.2.2.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
james_1  | WARNING: Please consider reporting this to the maintainers of com.google.inject.internal.cglib.core.$ReflectUtils$1
james_1  | WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
james_1  | WARNING: All illegal access operations will be denied in a future release
james_1  | 19:37:10.461 [INFO ] o.a.j.StartUpChecksPerformer - StartUpChecks all succeeded
james_1  | 19:37:10.583 [INFO ] o.a.j.CONFIGURATION - Load configuration file /opt/apache-james/james-server/conf/healthcheck.properties
james_1  | 19:37:10.670 [INFO ] o.a.c.b.FluentPropertyBeanIntrospector - Error when creating PropertyDescriptor for public final void org.apache.commons.configuration2.AbstractConfiguration.setProperty(java.lang.String,java.lang.Object)! Ignoring this property.
james_1  | 19:37:10.691 [INFO ] o.a.j.CONFIGURATION - Load configuration file /opt/apache-james/james-server/conf/james-database.properties
james_1  | 34  Global  WARN   [main] openjpa.Runtime - Could not create the optional validation provider. Reason returned: "A default ValidatorFactory could not be created."
james_1  | 19:37:11.049 [INFO ] o.a.j.CONFIGURATION - Load configuration file /opt/apache-james/james-server/conf/extensions.properties
james_1  | 19:37:11.093 [INFO ] o.a.c.b.FluentPropertyBeanIntrospector - Error when creating PropertyDescriptor for public final void org.apache.commons.configuration2.AbstractConfiguration.setProperty(java.lang.String,java.lang.Object)! Ignoring this property.
```